kilolib.c

```diff
@@ -505,6 +696,10 @@ ISR(TIMER0_COMPA_vect) {
     OCR0A = tx_increment;
     kilo_ticks++;
 
+    // Increments counter_ticks_for_voltage by 1 tick until 255
+    if(kilo_state == RUNNING && counter_ticks_for_voltage < 255)
+        counter_ticks_for_voltage++;
```

Comment:
this seems like a strange place to be fiddling with voltage. users are not guaranteed to call estimate_Distance, so relying on it will be flaky.